### PR TITLE
Feature Grant Creator TelegramID

### LINKED ISSUE #7688: Feature Grant/telegram
### I want to reach out the owner of a grant quickly, via telegram.

 (As a community manager, I want to ping grant owners for general reachouts.)

### Description

Asking grant creator for telegram id, which should be visible on grant page later. So, reaching out to grant creator is easy.
*Type*:

### PR DESCRIPTION
**Pull request by aamustapha**
##### Description.
As a community manager, I want to ping grant owners for general reachouts. Asking grant creator for telegram id, which should be visible on grant page later. So, reaching out to grant creator is easy.

##### Refers/Fixes
#7688

##### Testing
Loom: https://www.loom.com/share/b5a1751a1371495f8a86322fdb38e089

### CODE REVIEW

**Comment by thelostone-mc** [comment]
codewise LGTM  but left a comment on the ticket for product to answer 🙌 
https://github.com/gitcoinco/web/issues/7688#issuecomment-722150497

**Review by thelostone-mc — APPROVED** [review]

**Review by danlipert — CHANGES_REQUESTED** [review]
Code looks great, PR looks great, demo video looks great!

One thing - lets make it an optional field instead of required.

Great work!

File: app/grants/models.py [hunk 1]
```diff
@@ -182,6 +182,7 @@ class Meta:
     active = models.BooleanField(default=True, help_text=_('Whether or not the Grant is active.'))
     grant_type = models.ForeignKey(GrantType, on_delete=models.CASCADE, null=True, help_text="Grant Type")
     title = models.CharField(default='', max_length=255, help_text=_('The title of the Grant.'))
+    telegram_id = models.CharField(default='', max_length=255, help_text=_('The telegram id of Grant creator.'), blank=True, null=True)
```
Comment (thelostone-mc) [inline app/grants/models.py:185]: i think we can remove default value ! this is a nullable file after all

Comment (danlipert) [reply app/grants/models.py:185]: ah yes, nice find @thelostone-mc definitely agree

File: app/grants/templates/grants/detail/info.html [hunk 2]
```diff
@@ -119,6 +119,17 @@ <h1 class="font-title-xl my-4 font-weight-bold">
...
+        <div class="font-body mb-2">
+          <i style="width: 14px;" class="fab fa-telegram mr-2 text-center"></i>
+          <p class="d-inline-block mb-0">
+            <a href="https://t.me/{{ grant.telegram_id }}" target="_blank" rel="" data-toggle="tooltip" data-html="true" data-placement="top" for="telegram_id"
+            title="Creator's telegram ID">
+              {{ grant.telegram_id }}
+            </a>
+          </p>
+         </div>
+        {% endif %}
```
Comment (octavioamu) [inline app/grants/templates/grants/detail/info.html:132]: Isn't it for the whitelabel actually? cc @thelostone-mc 
I believe the request came from matic and they use the whitelabel form

**Review by octavioamu — CHANGES_REQUESTED** [review]

File: app/grants/templates/grants/detail/info.html (hunk 2 above)
Comment (thelostone-mc) [reply app/grants/templates/grants/detail/info.html:132]: I do know we intend to retire the white label form sooner or later but you are right !
cc @thesachinmittal on which form should this be available ?
/grants/new or /grants/matic/new  or both  ?

Comment (danlipert) [reply app/grants/templates/grants/detail/info.html:132]: Oh yeah that is true! Good catch @octavioamu !

@aamustapha - can you add the form input to `app/grants/templates/grants/new-whitelabel.html` as well? Thats another way grants get created but its specifically styled for Matic. Hopefully we can unify these two templates soon... but for now we have to remember to make the changes in both places

Comment (aamustapha) [reply app/grants/templates/grants/detail/info.html:132]: Sure

**Review by chibie — APPROVED** [review]
lgtm! only remember to add the form to new-whitelabel as @danlipert requested

**Comment by danlipert** [comment]
marking as draft until the `new-whitelabel` change is added

**Comment by danlipert** [comment]
@aamustapha also note there is a conflict you'll need to resolve

**Comment by aamustapha** [comment]
@thelostone-mc @danlipert @octavioamu can you please take a look at this.

**Comment by thelostone-mc** [comment]
@aamustapha I'm closing this out for now ! Will keep branch alive and we'll get it in if this becomes a requirement which is really needed

**Pull request closed without merging**